kuma/wiki/views.py

```diff
@@ -929,17 +932,18 @@ def new_document(request):
 
         allow_add_attachment = (
             Attachment.objects.allow_add_attachment_by(request.user))
-        return render(request, 'wiki/new_document.html',
-                            {'is_template': is_template,
-                             'parent_slug': parent_slug,
-                             'parent_id': initial_parent_id,
-                             'document_form': doc_form,
-                             'revision_form': rev_form,
-                             'WIKI_DOCUMENT_TAG_SUGGESTIONS': constance.config.WIKI_DOCUMENT_TAG_SUGGESTIONS,
-                             'initial_tags': initial_tags,
-                             'allow_add_attachment': allow_add_attachment,
-                             'attachment_form': AttachmentRevisionForm(),
-                             'parent_path': parent_path})
+        return render(
```

Comment:
Could you move the definition of the context to a variable? that would make this indentation a lot more readable, I've cleaned this up in some places here and there